test/utils.py

```diff
@@ -214,11 +214,15 @@
             }
         }
     },
-    "config": {}
+    "config": {
+        "develop": {
+            "data_timeout": 1.0
```

Comment:
To be on the safer side I'd rather make this change just in the `test_configure_bad_rx_device_status` test, rather than changing it for all tests.